test/puppetlabs/ring_middleware/core_test.clj

```diff
@@ -628,8 +628,13 @@
               response (stack (basic-request))
               json-body (json/parse-string (response :body))]
           (is (= 500 (response :status)))
-          (is (logged? #".*Internal Server Error.*" :error))
-          (is (re-matches #"Internal Server Error.*" (get json-body "msg" ""))))))
+          ;; Not the (?s) uses Java's DOTALL matching mode, so that a "." will match newlines
```

Comment:
"Note the..."?

Reply:
`(?s)` is the actual way that clojure adds Java match modes to a regex literal. I can quote the value and/or include a link to these docs if that would help:
https://clojuredocs.org/clojure.core/re-pattern#example-542692c6c026201cdc3268c7

Reply:
Yeah, good catch.